indra/llcommon/llapp.cpp

```diff
@@ -414,6 +414,22 @@ void LLApp::writeMiniDump()
 {
 }
 
+// static
+void LLApp::quit(bool force)
+{
+    if (LLApp* app = instance())
+    {
+        if (force)
+        {
+            app->forceQuit(); // Puts the viewer into 'shutting down without error' mode.
```

Comment:
If you are doing _exit to not report to bugsplat, you also need to remove SecondLife.exec_marker and logout_marker. Otherwise viewer will report "previous viewer session crashed" to statiscs server and we will have a hard time tracing where those crash-percents are comming from. Or alternatively provide a way for statistics to distinguish out of memory from everything else.

Reply:
LLAppViewer::removeMarkerFiles() is being called from ~LLAppViewer

Reply:
Ok, thanks for checking.